src/update-feeds-loops/get-updatable-feeds.ts

```diff
@@ -56,19 +57,26 @@ export const getUpdatableFeeds = (
         const newBeaconSetValue = calculateMedian(beaconValues.map(({ value }) => value));
         const newBeaconSetTimestamp = calculateMedian(beaconValues.map(({ timestamp }) => timestamp));
 
-        const { decodedUpdateParameters, dataFeedValue, dataFeedTimestamp } = dataFeedInfo;
+        const { decodedUpdateParameters, dataFeedValue, dataFeedTimestamp, decodedDapiName, dataFeedId } = dataFeedInfo;
         const adjustedDeviationThresholdCoefficient = multiplyBigNumber(
           decodedUpdateParameters.deviationThresholdInPercentage,
           deviationThresholdCoefficient
         );
 
-        return isDataFeedUpdatable(
-          dataFeedValue,
-          dataFeedTimestamp,
-          newBeaconSetValue,
-          newBeaconSetTimestamp,
-          decodedUpdateParameters.heartbeatInterval,
-          adjustedDeviationThresholdCoefficient
+        return logger.runWithContext(
```

Comment:
Will provide more context to [these logs](https://api3.grafana.net/explore?schemaVersion=1&panes=%7B%22wxv%22:%7B%22datasource%22:%22grafanacloud-logs%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bapp%3D%5C%22airseeker-v2%5C%22%7D%20%7C%3D%20%60Deviation%20exceeded%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22grafanacloud-logs%22%7D,%22editorMode%22:%22code%22,%22maxLines%22:5%7D%5D,%22range%22:%7B%22from%22:%22now-24h%22,%22to%22:%22now%22%7D%7D%7D&orgId=1).